app/assets/stylesheets/application.scss

```diff
@@ -94,13 +86,15 @@ a {
 }
 
 a:link {
-color: #FF3300;
+  color: #FF3300;
 }
+
 a:visited {
-color: #CC0000;
+  color: #CC0000;
 }
+
 a:hover {
-color: #999999;
+  color: #999999;
```

Comment:
Color literals like `#999999` should only be used in variable declarations; they should be referred to via variable everywhere else.